AudioManagerUtilities/src/CAmCommonAPIWrapper.cpp

```diff
@@ -36,21 +36,47 @@ namespace am
 static CAmCommonAPIWrapper* pSingleCommonAPIInstance = NULL;
 
 
+bool timeoutToTimespec(const int64_t & localTimeout, timespec & pollTimeout)
+{    
+    if(CommonAPI::TIMEOUT_INFINITE == localTimeout)//dispatch never
+    {
+        return false;
+    }
+    else
+    {
+        if(CommonAPI::TIMEOUT_NONE==localTimeout)//dispatch immediately
```

Comment:
Spaces
`if (CommonAPI::TIMEOUT_NONE == logcalTimeout) // dispatch immediately`

Reply:
Yes.  More than one occurrence pointed out by Jens here, and I am adding a few comments.  So since it doesn't come naturally (to the writer of the code), would you perhaps suggest to them to use  a code formatter?  (See discussion under #24, which gives pointers and there is indeed a code style already defined).

Reply:
We will fix the codestyle separately after the two pull requests are done.

Reply:
... and when will the two PRs be done?  :)